release_notes/gitops-release-notes-1-19.adoc

```diff
@@ -0,0 +1,53 @@
+
+{gitops-title} is a declarative way to implement continuous deployment for cloud native applications. {gitops-title} ensures consistency in applications when you deploy them to different clusters in different environments, such as development, staging, and production. {gitops-title} helps you automate the following tasks:
+
+* Ensure that the clusters have similar states for configuration, monitoring, and storage.
+* Recover or recreate clusters from a known state.
+* Apply or revert configuration changes to multiple {OCP} clusters.
+* Associate templated configuration with different environments.
+* Promote applications across clusters, from staging to production.
+
+For an overview of {gitops-title}, see xref:../understanding_openshift_gitops/about-redhat-openshift-gitops.adoc#about-redhat-openshift-gitops[About {gitops-title}].
+
+[NOTE]
+====
+For additional information about the OpenShift {gitops-shortname} lifecycle and supported platforms, refer to the link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] and link:https://access.redhat.com/support/policy/updates/openshift[Red{nbsp}Hat {OCP} Life Cycle Policy].
+====
+
+// Compatibility and support matrix
+include::modules/go-compatibility-and-support-matrix.adoc[leveloffset=+1]
+
+// Modules included, most to least recent
+
+// Release notes for Red Hat OpenShift GitOps 1.19.0
+include::modules/gitops-release-notes-1-19-0.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+
+* xref:../argo_cd_agent_architecture/argocd-agent-architecture-overview.adoc[Argo CD Agent architecture overview]
+
+* xref:../argocd_agent_installation/argocd-agent-installation.adoc[Installing Argo CD Agent]
+
+* xref:../argocd_instance/setting-up-argocd-instance.adoc#gitops-configuring-imagepullpolicy_setting-up-argocd-instance[Configuring image pull policy]
+
+* xref:../managing_resource/configure-resource-requests-and-limits-for-gitops-plugin-components.adoc[Configure resource requests and limits for GitOps plugin components]
+
+* xref:../argocd_instance/argo-cd-cr-component-properties.adoc#gitops-configuring-notifications-any-namespace_argo-cd-cr-component-properties[Configuring Notifications in any Namespace]
+
+* link:https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.14-3.0/#use-annotation-based-tracking-by-default[Annotation-based resource tracking, Argo CD upstream documentation]
+
+* link:https://argocd-image-updater.readthedocs.io/en/stable/#argo-cd-image-updater[Argo CD Image Updater, Argo CD upstream documentation]
+
+* link:https://developers.redhat.com/content-gateway/rest/browse/pub/openshift-v4/clients/openshift-gitops/[Red Hat Content Gateway binaries, upstream location]
```

Comment:
```suggestion
* link:https://developers.redhat.com/content-gateway/rest/browse/pub/openshift-v4/clients/openshift-gitops/[Red Hat OpenShift GitOps client binaries]
```
Can we change the title to drop upstream? The binaries are stored in RedHat's content gateway.